test/OpenTelemetry.Contrib.Shared.Tests/OpenTelemetry.Contrib.Shared.Tests.csproj

```
@@ -31,6 +31,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EmbeddedResource Include="SqlProcessorAdditionalTestCases.json">
+      <LogicalName>SqlProcessorAdditionalTestCases.json</LogicalName>
+    </EmbeddedResource>
     <EmbeddedResource Include="SqlProcessorTestCases.json">
       <LogicalName>SqlProcessorTestCases.json</LogicalName>
     </EmbeddedResource>
```

These could be simplified down to:

```suggestion
    <EmbeddedResource Include="*.json" LogicalName="%(FileName)%(Extension)" />
```